users/miketronic/rules.mk

```diff
@@ -0,0 +1,91 @@
+SRC += miketronic.c tapdances.c process_records.c
+
+
+# Common feature for all keyboards
+BOOTMAGIC_ENABLE  = yes
+EXTRAKEY_ENABLE   = yes
+TAP_DANCE_ENABLE  = yes
+MACROS_ENABLED    = yes
+RGBLIGHT_ENABLE   = no
+RGB_MATRIX_ENABLE = no
+
+# idobao/id75
+ifeq ($(strip $(KEYBOARD)), idobao/id75/v1)
```

Comment:
these should be left at the kb level,

Reply:
All of them?  So I should leave the rules.mk for each keyboard in the keymap folder? 
 I.e. ...pi40/keymaps/miketronic/rules.mk

Reply:
> All of them, so they shouldn't stay centralized? So I should leave the rules.mk for each keyboard in their keymap folder? I.e. ...pi40/keymaps/miketronic/rules.mk

Nevermind I see it now on other userspaces. Will correct.